components/brave_rewards/resources/tip/components/app_error.tsx

```diff
@@ -23,11 +34,7 @@ export function AppError (props: Props) {
     <style.root>
       <style.graphic className={isConnectionError ? 'server-error' : ''} />
```

Comment:
We might want to show the "server-error" graphic in the tip failure case, at least until we have a separate image. Actually, that image might work for this use case.